### PR TITLE
Remove expand-methods in favor of deftype macro

### PR DESCRIPTION
**Pull request by patrickgombert**

### CODE REVIEW

File: src/clj/clojure/lang/apersistent_map.clj [hunk 1]
```diff
@@ -48,4 +44,4 @@
           (let [entry (first s)]
             (recur (assoc mp (key entry) (val entry)) (next s)))
           mp))))
- 
+
```
Comment (mylesmegyesi) [inline src/clj/clojure/lang/apersistent_map.clj]: :astonished: :space_invader:

Comment (patrickgombert) [inline src/clj/clojure/lang/apersistent_map.clj]: ?

Comment (mylesmegyesi) [inline src/clj/clojure/lang/apersistent_map.clj]: You added a whitespace to the end of the file.

**Comment by patrickgombert** [comment]
removed whitespace

**Pull request merged**